src/sass/rx/_partials/_rx-prescriptions.scss

```diff
@@ -102,6 +102,15 @@ button.rx-prescription-button,
   position: relative;
 }
 
+.rx-prescription-status,
+.rx-prescription-refill-requested {
+  font-weight: bold;
+}
+
+.rx-prescription-status {
+  text-transform: capitalize;
```

Comment:
Doesn't seem necessary if you're already translating it to a capitalized form with `rxStatuses`.